misk/src/main/kotlin/misk/web/actions/NotFoundAction.kt

```diff
@@ -38,4 +50,40 @@ class NotFoundAction @Inject constructor() : WebAction {
       )
     }
   }
+
+  /** Generates a message for a client that hints what may have been wrong with their request. */
+  private fun notFoundMessage(path: String, httpCall: HttpCall) = buildString {
+    val boundActions = servletProvider.get().boundActions
+    val alternativeActions = boundActions.mapNotNull {
+      it.matchByUrl(httpCall.url)
+    }.filterNot { it.action.pathPattern.matchesWildcardPath }
+
+    append("Nothing found at /$path.\n\n")
+    append("Received:\n")
+    append("  ${httpCall.dispatchMechanism.method} /$path\n")
```

Comment:
Consider an explicit indentation character like `\t` or the following which only need to be set once vs. manually adding spaces.

- Increase indent: `⇥`
- Decrease indent: `⇤`

Reply:
nooo tabs are bad!

Reply:
Agreed 💯 . No indentation for the following lines would also be totally valid too!

Reply:
Ah I just updated this to use tabs! What if I did `val INDENT = "  "` and used that everywhere? Would that be an okay solution?

Reply:
I'd lean towards just not having an indent at all

Reply:
Removed the indentation. No tabs, no spaces, just info :)